### ISSUE TITLE
[Sober] Block app bundles by checksum on Sober

### ISSUE THREAD

**Issue by alejandro-alzate** [vinegarhq/sober#198]
### Before Creating the Enhancement Request

- [X] I have confirmed that this should be classified as an enhancement rather than a bug/feature.

### Is your feature request related to a problem?

Yes. Hackers. they ruin our party.
And keep sober closed source until roblox wakes up and notices that Linux is a thing instead of turning a blind eye.

### Describe the Solution you'd Like

I noticed that sober didn't accepted the latest roblox apk bundle although this seems like a technical limitation, I want that to become it's strength.

Check the bundle with a checksum list so hackers CANNOT poke at it and make Roblox mad.

Just making a protected checksum list (With the checksum of THAT list inside of the executable to put further roadblocks)
Where you know it is clean and most importantly **_unmodified_**.

Although my point of view is all software should be open source, I think this is a fair way to get the rotten apples OUT of the app.

Most people including me cherished this effort to getting what the lazy Roblox corp doesn't wanna do. (Because funny number must go up).

So in a nutshell intentionally block suspicious apk bundles
Because a gazillion dollar company feel like not doing their homework, and make a honest to god native client with proper anticheat...

### Describe Alternatives you've considered

Really I'm not sure I really think that the only way of stopping script kiddies is literally saying to them "No you don't!" (This apk bundle is not compatible).

### Additional context

Sorry in advance for posting here. Since the project does not have GitHub (and maybe it should be maintained like this) this is the only place that I know that my humble request would get somewhere ...

You can close it immediately if you want. I can't even stop you if I wanted, but at least keep it on the back of your mind...

**Comment by apprehensions** [comment]
It has a GitHub.

**Comment by lucasmz-dev** [comment]
Sober does already check app signatures AFAIK; which is also beneficial to the security of the user
As long as Sober also enforces a minimal version, then that already achieves blocking alternative clients

**Comment by alejandro-alzate** [comment]
> It has a GitHub.

Uh I tried to google it I thought that didn't had
Is google shadowing it?

Next time I'll check directly the github page.
This is why I use ddg I swear, but I have no explanation why I googled it instead.

**Comment by alejandro-alzate** [comment]
> _Sober does already check app signatures_

So you can close this one then if you want.

Also latest version didn't worked on it (`com_roblox_client_2_640_730_1670_3arch_[...].apkm`)

Is this a technical limitation?

I would say
_"What makes Sober ticks?"_
But is better if I say for sake of longevity of the project 
_"Alright keep your secrets then"_

> As long as Sober also enforces a minimal version code

But this is a clue that it only checks the version. Or do you have checksum checks in place?
That's the thing I think this will be the next step to keep cheaters out (At least the script kiddo kind)...
